test/fixtures/comparison-files/lib/viewers/ResultsViewer.test.ts/one-multilocation-violation-details.txt

```diff
@@ -5,6 +5,6 @@
     message:   This is a message
     locations:
         __PATH_TO_FILE_A__:20:1
-        (main) __PATH_TO_FILE_Z__:2:1 This is a comment at Location 2
-        __PATH_TO_FILE_A__:1:1 This is a comment at Location 3
-    resources: https://example.com/stub1RuleA
+        (main) __PATH_TO_FILE_Z__:2:1 "This is a comment at Location 2"
+        __PATH_TO_FILE_A__ (1:1-3:1) "This is a comment at Location 3"
```

Comment:
Using the same way of displaying a range as the html report does. But kept the original style when there is no endLine so that we maintain the clickability in VSCode whenever possible.

Alternatively we could abandon the nice to have of having the `file:startLine:startColumn` clickable and we could just display `file (startLine:startColumn)` like the html report does always. Thoughts?